### ISSUE TITLE
Preview clip encode not completing

### ISSUE THREAD

**Issue by iamjaydear** [RandomEngy/VidCoder#109]
Currently using VC 2.32 beta x64. Preview clip encode permanently stalls near completion. If I try to cancel it VC locks up and has to be killed. Same happens if instead I try to exit the preview window. If I don't do these, then the main program seems to continue to be responsive and functional, albeit with a frozen preview window. Log is attached.
I've tried selecting different previews and different preview lengths with same outcome. Everything else seems OK - good work BTW :*
[PreviewCrashLog.txt](https://github.com/RandomEngy/VidCoder/files/395333/PreviewCrashLog.txt)

**Comment by RandomEngy** [comment]
Does the latest HandBrake nightly have the same behavior?

**Comment by iamjaydear** [comment]
Well... I installed v20160730184204-9b7ef42. I then selected a profile and clicked preview. The still frame looked normal except the box center bottom has a button called Live Preview which when pressed says it can't find the preview file or it was deleted. There isn't any other way to make a preview file that I can see, so I'd say it's completely screwy. Why "Live" preview I wonder, when there's no other kind of preview available - got me! It's a good reminder of why we use VC.
